package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-frost-core",
-  "version": "0.22.2",
+  "version": "0.22.3",
```

Comment:
Dont' manually bump the version as the CI will take care of this if/when merged.